source/infrastructure/jenkins-key-expired.erb

```diff
@@ -0,0 +1,16 @@
+---
+title: How to renew the Jenkins GPG Key
+weight: 100
+last_reviewed_on: 2021-25-01
+review_in: 6 months
+---
+
+# How To Renew The Jenkins GPG Key
+
+If you see an error stating that this key has expired, please contact the SRE
+team.
+
```

Comment:
Would it be possible to add the info where these errors are appearing?

Reply:
Hey Seb

I've added a sentence on where the errors appear.